arches/app/views/graph.py

```diff
@@ -210,23 +225,15 @@ def get(self, request, graphid):
         if self.graph.ontology is not None:
             branch_graphs = branch_graphs.filter(ontology=self.graph.ontology)
```

Comment:
1. Create a branch from scratch (in graph designer)
2. Save Edits
3. Quit Designer
4. [You're on Branches tab of Graph Designer:] Click on that branch


Raises

```py
  File "/Users/jwalls/prj/arches/arches/app/views/graph.py", line 225, in get
    if self.graph.ontology is not None:
       ^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'ontology'
```

Reply:
I also cannot seem to repro this either. Happy to chat if you wanna show off your workflow && compare